lib/logstash/plugin_mixins/jdbc/value_tracking.rb

```diff
@@ -5,28 +5,26 @@ module LogStash module PluginMixins module Jdbc
   class ValueTracking
 
     def self.build_last_value_tracker(plugin)
+      handler = NullFileHandler.new(plugin.last_run_metadata_path)
```

Comment:
Could we do this as a one-liner to avoid unnecessary creation of `NullFileHandler`?

```suggestion
      handler = plugin.record_last_run ? FileHandler.new(plugin.last_run_metadata_path) : NullFileHandler.new(plugin.last_run_metadata_path)
```